docs/howtoguides/enable_landscape.rst

```diff
@@ -0,0 +1,47 @@
+How to enable Landscape
+***********************
+
+``pro`` offers a convenient way to register a machine with Landscape via the
+``pro enable landscape`` command.
+
+Enable interactively
+====================
+
+To register your machine by interactively providing your Landscape account
+details at the CLI, run:
+
+.. code:: bash
+
+   sudo pro enable landscape
+
+That command will install ``landscape-client`` and drop you into an interactive
+wizard to complete the Landscape registration for the machine.
+
+Enable non-interactively
+========================
+
+If you know the details of your Landscape setup and would like to register a
+machine without any interaction necessary, you can pass ``landscape-config``
+parameters to ``pro enable landscape`` after a ``--``. You should also use the
+``--assume-yes`` flag to ensure there are no prompts.
+
+Altogether, it should look something like this:
+
+.. code:: bash
+
+   sudo pro enable landscape --assume-yes -- --account-name my-account --computer-title my-computer
```

Comment:
```suggestion
The command to enable Landscape takes the following format:

.. code:: bash

   sudo pro enable landscape -- <landscape-config parameters>

Which, when the parameters are added, should look something like this:

.. code:: bash

   sudo pro enable landscape --assume-yes -- --account-name <my-account> --computer-title <my-computer>
```

Reply:
Also, do we need to list all the (currently) possible parameters?

Reply:
I'll make that change, and I posted above a link that I'll include which describes the possible parameters